locales/en.yml

```diff
@@ -75,7 +75,9 @@ en:
 
           Service:
               A service provided by sccli or systemd. For example:
-               docker, k8s, or openshift etc.
+               sccli service like docker, kubernetes, openshift etc
+               or
+               systemd service like libvirtd etc
```

Comment:
so we are going ahead to control any systemd service now? Has this been discussed? This is imo not consistent with for example `vagrant service-manager status`. -1 from me. Was the plan not to discuss this first on the mailing list?